korio/src/commonTest/kotlin/korlibs/io/serialization/xml/XmlTest.kt

```diff
@@ -153,4 +153,11 @@ class XmlTest {
             xml.take(10).toList().joinToString("\n")
         )
     }
+
+    @Test
+    fun testEntities() {
```

Comment:
Could we use a more descriptive name that clearly indicates what we're testing in this case?